app/views/mailer/notify.html.erb

```diff
@@ -4,4 +4,16 @@ Dear <%= @contact[:name] %>,
 
 Here is all the information they recorded about their trip:
 
+Trip starting point: <%= @trip[:startingPoint]%>
+Trip ending point: <%= @trip[:endingPoint]%>
+Trip ending point: <%= @trip[:endingPoint]%>
+Number of traveling companions: <%= @trip[:travelingCompanions] %>
+
+Gear: <%= @trip[:tripGears][0] %>
+
+Vehicle description: <%= @trip[:vehicle] %>
+
+Physical description: <%= @trip[:user] %>
+
 Please contact a nearby search and rescue team below:
+<%= @trip[:nearbySarTeams] %>
```

Comment:
Was this displaying all nearby teams?  I was thinking we would have to iterate of nearbySarTeams

Reply:
I thought this was how it was formatted. There was no way for me to get a real call back with how the info looked. I was following the other guides. Let me know how you want to deal with it...